tests/framework/installer/ceph_installer.go

```diff
@@ -425,12 +425,14 @@ func (h *CephInstaller) InstallRook(namespace, storeType string, usePVC bool, st
 	}
 
 	// Create rook cluster
-	err = h.CreateRookCluster(namespace, onamespace, storeType, usePVC, storageClassName,
-		cephv1.MonSpec{Count: mon.Count, AllowMultiplePerNode: mon.AllowMultiplePerNode}, startWithAllNodes,
-		skipOSDCreation, h.CephVersion)
-	if err != nil {
-		logger.Errorf("Rook cluster %s not installed, error -> %v", namespace, err)
-		return false, err
+	for _, namespace := range clusterNamespaces {
```

Comment:
The tests don't need to support multiple namespaces for the non-helm case, except for the multi-cluster test suite, which is already implemented separately.